bazel/external/wee8.genrule_cmd

```diff
@@ -2,11 +2,14 @@
 
 set -e
 
-# This works only on Linux-x86_64 and macOS-x86_64.
-if [[ ( `uname` != "Linux" && `uname` != "Darwin" ) || `uname -m` != "x86_64" ]]; then
-  echo "ERROR: wee8 is currently supported only on Linux-x86_64 and macOS-x86_64."
+# This works only on Linux-{x86_64,s390,s390x} and macOS-x86_64.
```

Comment:
This comment doesn't match what it does below? where is s390?

Reply:
Sorry, the comment is wrong. s390 is a 32-bit variant of IBM Z, and in practice it's barely used. So I will push a new version with a more accurate comment.